### DISCUSSION TITLE
How to disable this white area of sidebery?

### DISCUSSION THREAD

**Discussion by 1human** [mbnuqw/sidebery#1562] [General]
A white area  appeared after I updated Firefox to 107.0b2
![Snipaste_2022-10-19_19-13-36](https://user-images.githubusercontent.com/18576031/196676793-fb34b79e-cd35-4fd7-9cb7-e38e85dc03ef.png)
209--539px on my laptop
Sidebery version 5.0.0b27
Should I waiting for the Firefox to fix this?
Thanks!

**Comment by mbnuqw** [comment]
Cannot reproduce this on 107.0b2, maybe it's some custom Sidebery CSS or/and userChrome?

**Comment by dexeonify** [comment]
Sounds like [bug 1792277](https://bugzilla.mozilla.org/show_bug.cgi?id=1792277). Nightly is on 108.0a1 while Beta is on 107.0b3, so update your Firefox.

**Comment by 1human** [comment]
> Cannot reproduce this on 107.0b2, maybe it's some custom Sidebery CSS or/and userChrome?

Yes...  I used a custom css file from reddit which can provide an experience like edge...
Looks like the sidebar is not fully folded

> 


#main-window[tabsintitlebar="true"]:not([extradragspace="true"]) #TabsToolbar > .toolbar-items {
  opacity: 0;
  pointer-events: none;
}

#main-window:not([tabsintitlebar="true"]) #TabsToolbar {
    visibility: collapse !important;
}
 
#nav-bar {
    margin-right:140px;
}
 
/* Some space on top to allow window dragging */
#titlebar {
    appearance: none !important;
    height: 4px;
}
 
/* Fix for main menu calling by Alt button */
#titlebar > #toolbar-menubar {
    margin-top: 10px;
}
 
.titlebar-buttonbox-container {
  display: block;
  height: 35px;
  margin-top: -2px;
}
 
.titlebar-buttonbox-container .titlebar-button {
  display: block;
  height: 40px;
}
 
:root {
  --sidebar-min-width: 20px;
  --sidebar-visible-width: 210px;
}

#sidebar-header {
  overflow: hidden !important;
}

#sidebar-box #sidebar-header {
  display: none !important;
}

#sidebar,
#sidebar-header {
  position: relative !important;
  min-width: var(--sidebar-min-width) !important;
  max-width: var(--sidebar-min-width) !important;
transition: .1s ease .05s;
  z-index:1;
}

#sidebar-box:hover :-moz-any(#sidebar,#sidebar-header) {
  background-color: var(--toolbar-bgcolor) !important;
  min-width: var(--sidebar-visible-width) !important;
  max-width: var(--sidebar-visible-width) !important;
  margin-right: calc((var(--sidebar-visible-width) - var(--sidebar-min-width)) * -1) !important;
  z-index:1;
  position: relative !important;
transition: .1s ease .05s;

}

**Comment by mbnuqw** [comment]
I've checked the userChrome, you just need to manually decrease width of the static sidebar or increase `--sidebar-visible-width` in your CSS.
